sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultDataStore.java

```diff
@@ -1042,29 +1045,25 @@ private String getMetricName(String name) {
         return MetricRegistry.name("bv.emodb.sor", "DefaultDataStore", name);
     }
 
-    private Iterable<UpdateRef> convertToUpdateRef(Iterable<UpdateRefModel> apiUpdateRefs) {
-        List<com.bazaarvoice.emodb.sor.core.UpdateRef> coreUpdateRefs = new ArrayList<>();
-        for (UpdateRefModel apiUpdateRefModel : apiUpdateRefs) {
-            String tableName = apiUpdateRefModel.getTable();
-            String key = apiUpdateRefModel.getKey();
-            UUID changeId = apiUpdateRefModel.getChangeId();
-            Set<String> tags = apiUpdateRefModel.getTags();
-            coreUpdateRefs.add(new com.bazaarvoice.emodb.sor.core.UpdateRef(tableName, key, changeId, tags));
-        }
-        return coreUpdateRefs;
-    }
-
     @Override
-    public void updateRefInDatabus(Iterable<UpdateRefModel> updateRefs, Set<String> tags, boolean isFacade) {
-        Iterator<UpdateRef> updateRefsIter = convertToUpdateRef(updateRefs).iterator();
-        if (!updateRefsIter.hasNext()) {
-            return;
-        }
-        List<UpdateRef> updateRefList = new ArrayList<>();
-        while (updateRefsIter.hasNext()) {
-            UpdateRef updateRef = updateRefsIter.next();
-            updateRefList.add(updateRef);
+    public void updateRefInDatabus(List<String> updateRefsModel) {
+        try {
+            List<UpdateRef> updateRefModelList = updateRefsModel.stream()
+                    .map(string -> {
+                        try {
+                            return new ObjectMapper().readValue(string, UpdateRef.class);
+                        } catch (JsonProcessingException e) {
+                            _log.error("Error In Parsing The Message: " , e);
```

Comment:
log the 'string'

Reply:
Resolved in this [pr](https://github.com/bazaarvoice/emodb/pull/851)